docker/images/pinot/etc/jmx_prometheus_javaagent/configs/minion.yml

```diff
@@ -1,4 +1,22 @@
 rules:
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"MinionMetrics\", name=\"pinot.minion.version.(\\w+)\"><>(\\w+)"
```

Comment:
move this rule here so that it will take effects before more generic ones.

Reply:
discussed offline.